rq/worker.py

```diff
@@ -625,15 +625,24 @@ def monitor_work_horse(self, job):
         either executes successfully or the status of the job is set to
         failed
         """
+
+        ret_val = None
         while True:
             try:
+                job.started_at = job.started_at or utcnow()
```

Comment:
`job.started_at` should be set outside of the `while` loop, correct? Otherwise `job.started_at` will keep increasing everytime the parent checks whether horse is still alive.

Reply:
I'll test this, but I think that the `or` will only fire the `utcnow()` method once because `job.started_at` is `None`. After this, its a useless assignment. I'll move it outside the loop anyways because its better form. Thanks for the review.